src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -486,6 +512,34 @@ class ExplorerButtonGroup extends React.Component {
     const dropdownConfigs = calculateDropdownButtonConfigs(this.props.buttonConfig);
     return (
       <React.Fragment>
+        {
+          // REMOVE THIS CODE WHEN EXPORT TO TERRA WORKS
+          // ===========================================
+          this.state.enableTerraWarningPopup &&
+            (<Popup
+              message={terraExportWarning.message
+                ? terraExportWarning.message
+                : `Warning: You have selected more subjects than are currently supported. The import may not succeed. Terra recommends slicing your data into segments of no more than ${terraExportWarning.subjectThreshold.toLocaleString()} subjects and exporting each separately. Would you like to continue anyway?`
```

Comment:
should we make the word `subjects` here configurable like this https://github.com/uc-cdis/data-portal/blob/master/src/GuppyDataExplorer/ExplorerTopMessageBanner/index.jsx#L41-L42 ?

Reply:
this is fine sine the message is configurable in gitops.json, closing this 😃